src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -22,6 +22,8 @@ public sealed class ActorRuntime
     {
         // Map of ActorType --> ActorManager.
         private readonly Dictionary<string, ActorManager> actorManagers = new Dictionary<string, ActorManager>();
+
+        private static Dictionary<string, IDaprInteractor> daprInteractors = new Dictionary<string, IDaprInteractor>();
```

Comment:
I think we can find a solution that doesn't require a static look up. I'm not sure why this had to be added.

Some general feedback about this design:
- When you think static, think **global variable** - if you're making something mutable and static, that's a red flag. If you're making a mutable static collection, then that's a double red flag.
- Dictionary is not thread safe so this kind of pattern will lead to all kinds of wierd behaviors and you won't catch it in tests